run.py

```diff
@@ -106,6 +114,26 @@ def config_validator():
         app.config['TESTING'] = True
         print("Starting server in ~*TEST MODE*~")
         app.run(host='localhost')
+    elif args.backup:
+        issue_engine = create_engine('sqlite:///' + os.path.join(app.config['BASE_DIR'], 'issues.db'))
+        issue_session_maker = sessionmaker(autocommit=False,
+                                           autoflush=False,
+                                           bind=issue_engine)
+        issue_db = scoped_session(issue_session_maker)
+        # Take a backup if issues.db has data dump.
+        # TODO: write code for creating issues dump in the issues.db with updated schema
+        if issue_db().execute('select count(*) from webcompat_issues').fetchall()[0][0] > 0:
+            if not os.path.exists(app.config['BACKUP_DEFAULT_DEST']):
+                print 'Creating backup directory'
```

Comment:
Might be kinda cool to do something like "Creating backup directory at _____BACKUP_DEST___whatever" (or similar).